docs/standard/modelling/repr-state-owned-enterprises.rst

```diff
@@ -21,92 +21,76 @@ In particular, the following properties and values SHOULD be used to represent i
 
 For more details about these properties see :ref:`schema-entity-record`.
 
-A number of scenarios are outlined below. We show how the these properties and others can be used to represent information about SOEs within a set of connected BODS statements. 
+Examples
+--------
 
+Ownership by state body
+^^^^^^^^^^^^^^^^^^^^^^^
 
-Example scenarios
-------------------------
-
-These examples show how statements might be connected to represent the involvement of a state with an enterprise. Only those property:value pairs that are particularly relevant to the scenario are shown in the statement blocks. Complete data would also show, for instance, the particular types of interests held by the state, such as an 80% shareholding. 
-
-
-Scenario 1
-^^^^^^^^^^
-
-Ownership-or-control by a state is known and a statement about ownership is possible.
+Beneficial ownership by a state body is known and a statement about ownership is possible.
```

Comment:
I forgot to explain this. It's because the concept of 'beneficial ownership' doesn't neatly apply to states. In most jurisdictions, a state or state body cannot be a beneficial owner.

Reply:
ahh interesting!